scripts/cleanup.py

```diff
@@ -0,0 +1,12 @@
+import os
+import time
+
+MEDIA_DIR = './src/app/media/AnonymousUser'
+DAYS_THRESHOLD = 10
+
+now = time.time()
+
+for filename in os.listdir(MEDIA_DIR):
```

Comment:
Let's add a logger to track time of cron job and also log the files it's deleting and date of the file. Also, cron should log to that file by default.

Reply:
Ok, I will add that and update the PR